examples/Remarketing/AddFlightsFeed.php

```diff
@@ -0,0 +1,423 @@
+     */
+    private static function getFeed(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $feedResourceName
+    ) {
+        $googleAdsServiceClient = $googleAdsClient->getGoogleAdsServiceClient();
+        // Constructs the query to get the feed attributes for the specified feed resource name.
+        $query = "SELECT feed.attributes FROM feed WHERE feed.resource_name = '$feedResourceName'";
+        // Issues a search request by specifying page size.
+        $response =
+            $googleAdsServiceClient->search($customerId, $query, ['pageSize' => self::PAGE_SIZE]);
+
+        // Gets the first result because we only need the single feed we created previously.
+        /** @var GoogleAdsRow $googleAdsRow */
+        $googleAdsRow = $response->getIterator()->current();
+
+        // Gets the attributes list from the feed and creates a map with keys of each attribute and
+        // values of each corresponding ID.
+        $feedAttributes =
+            iterator_to_array($googleAdsRow->getFeed()->getAttributes()->getIterator());
+
+        $feedPlaceHolderFieldsList = array_map(function (FeedAttribute $feedAttribute) {
+            switch ($feedAttribute->getNameUnwrapped()) {
+                case 'Flight Description':
+                    return FlightPlaceholderField::FLIGHT_DESCRIPTION;
+                    break;
+                case 'Destination ID':
+                    return FlightPlaceholderField::DESTINATION_ID;
+                    break;
+                case 'Flight Price':
+                    return FlightPlaceholderField::FLIGHT_PRICE;
+                    break;
+                case 'Flight Sale Price':
+                    return FlightPlaceholderField::FLIGHT_SALE_PRICE;
+                    break;
+                case 'Final URLs':
+                    return FlightPlaceholderField::FINAL_URLS;
+                    break;
+                // See FlightPlaceholderField.php for all available values.
+                default:
+                    throw new \RuntimeException('Invalid feed attribute name.');
+            }
+        }, $feedAttributes);
+
+        return array_combine($feedPlaceHolderFieldsList, $feedAttributes);
+    }
+
+    /**
+     * Creates a feed mapping for a given feed.
+     *
+     * @param GoogleAdsClient $googleAdsClient the Google Ads API client
+     * @param int $customerId the customer ID
+     * @param string $feedResourceName the feed resource name for creating a feed mapping
+     * @param array $placeHoldersToFeedAttributesMap the map from placeholder fields to feed
+     *      attributes
+     */
+    private static function createFeedMapping(
+        GoogleAdsClient $googleAdsClient,
+        int $customerId,
+        string $feedResourceName,
+        array $placeHoldersToFeedAttributesMap
+    ) {
+        // Maps the feed attribute IDs to the field ID constants.
+        $flightDescriptionMapping = new AttributeFieldMapping([
```

Comment:
All these attribute field mapping creations could be factorized into a "createAttributeFieldMapping" method directly called from the feedMapping object creation

Reply:
Do you mean creating a private method for that?
I'm rather not nit-picky on this so we don't diverge much from the golden PR.
The current block is small enough to fit here with reasonable variable names used.
The part where all these variables are passed to FeedMapping creation also has reasonable readability as well.